tests/everflow/everflow_test_utilities.py

```diff
@@ -355,9 +356,9 @@ def setup_info(duthosts, rand_one_dut_hostname, tbinfo, request, topo_scenario):
     # If we send TTL=1 packet we don't need this but in multi-asic TTL > 1
 
     if 't2' in topo:
-        for duthost in duthosts.frontend_nodes:
-            duthost.command("sudo config bgp shutdown all")
-            duthost.command("mkdir -p {}".format(DUT_RUN_DIR))
+        for dut_host in duthosts.frontend_nodes:
```

Comment:
Why do we need this change?

Reply:
I changed the name local variable `duthost` because we have it defined at a higher level. I don't want to mix them

Reply:
hi @bingwang-ms if you want to change variable names, could you also change on line 373 when enabling BGPs as well? your previous commit only changed the variable when it's shutting down bgps.. or you can leave as is..

Reply:
Thanks! That's a good catch. I change the variable name in both Line 359 and Line 372.